index.html

```diff
@@ -0,0 +1,145 @@
+                    <h3>Sun & good wine land</h3>
+                    <h1>Cafayate</h1>
+                    
+                </div>
+                <div class="par-date" >
+                    <article>
+                        Cafayate is a town in the Calchaquíes valleys located in the southwest of the province of Salta, northwestern region of Argentina. It is the head of the department of Cafayate and recognized for the quality of the wines produced there.
+                    </article>
+                    <div class="date-place">
+                        <h2>2023.9.6(WED) ~ 8(FRI)</h2>
+                        <p>@National Technological University</p>
+                    </div>
+                </div>
+                
+            </div>
+        </main>
+    </section>
+    
+    
+    <!-- Activities -->
+    <section class="main-program">
+        <div class="mp-container">
+            <h2>Activities</h2>
+
+            <ul class="program-list">
+                <li class="program-item">
+                    <div class="act-title">
+                        <i class="fa-solid fa-person-hiking"></i>
+                        <h3>Trekking</h3>
+                    </div>
+                    <p>Explore majestic trails, immerse yourself in nature's wonders, and let adventure ignite your soul.</p>
+                </li>
+                <li class="program-item">
+                    <div class="act-title">
+                        <i class="fa-solid fa-bed"></i>
+                        <h3>Lodging</h3>
+                    </div>
+                    <p>Unwind and indulge in pure bliss, where comfort meets tranquility, your perfect lodging escape.</p>
+                </li>
+                <li class="program-item">
+                    <div class="act-title">
+                        <i class="fa-sharp fa-solid fa-utensils"></i>
+                        <h3>Gastronomy</h3>
+                    </div>
+                    <p>Savor exquisite flavors, where culinary artistry meets your palate, an epicurean delight awaits.</p>
+                </li>
+                <li class="program-item">
+                    <div class="act-title">
+                        <i class="fa-solid fa-wine-glass"></i>
+                        <h3>Wine Cellar</h3>
+                    </div>
+                    <p>Elevate your senses with wines that reach new heights, where excellence and altitude intertwine.</p>
+                </li>
+                <li class="program-item">
+                    <div class="act-title">
+                        <i class="fa-solid fa-road"></i>
+                        <h3>Guidenes</h3>
+                    </div>
+                    <p>Embark on immersive journeys, unravel hidden gems, and create unforgettable memories through our guided tours</p>
+                </li>
+            </ul>
+            <button class="act-btn"> See all activities</button>
+        </div>
+        
+    </section>
```

Comment:
- [ ] Please 🙏🏻  adhere to the project requirement according to which you can make little changes or adjustments of your own choice but can't change it completely but as I can see that your design looks different than actual, which makes your design `inconsistent` and looks unprofessional. Please have a look at the below-attached image and table.

### Requirement
![image](https://github.com/ulises2607/cayafate-project/assets/66476329/cd029716-65ab-45b5-8e7d-6886f05d48af)


### Desktop Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-22 16-23-30](https://github.com/ulises2607/cayafate-project/assets/66476329/1740fdb3-a789-41bb-a793-1b04a9133498)

</td>
<td>

![Screenshot from 2023-07-08 15-49-27](https://github.com/ulises2607/cayafate-project/assets/66476329/dbf05a6d-9534-4fb8-ab39-50284171cbb4)

</td>
</tr>
</table>

### Mobile Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![image](https://github.com/ulises2607/cayafate-project/assets/66476329/821d8e2b-b333-4385-b85e-c7daa1b6ddba)

</td>
<td>

![Screenshot from 2023-07-08 15-49-15](https://github.com/ulises2607/cayafate-project/assets/66476329/6f6482ed-8f7f-43c4-8cd5-db9ea65382cd)

</td>
</tr>
</table>